include/seqan3/range/view/kmer_hash.hpp

```diff
@@ -61,42 +177,51 @@ struct kmer_hash_fn
-     *                      omitted in pipe notation]
-     * \param[in] urange    The range being processed. [parameter is omitted in pipe notation]
-     * \returns             A range of unsigned integral values where each value is the hash of the resp. k-mer.
-     *                      See below for the properties of the returned range.
-     * \ingroup view
-     *
-     * ### View properties
-     *
-     * | range concepts and reference_t  | `urng_t` (underlying range type)      | `rrng_t` (returned range type)                     |
-     * |---------------------------------|:-------------------------------------:|:--------------------------------------------------:|
-     * | std::ranges::InputRange         | *required*                            | *preserved*                                        |
-     * | std::ranges::ForwardRange       | *required*                            | *preserved*                                        |
-     * | std::ranges::BidirectionalRange |                                       | *preserved*                                        |
-     * | std::ranges::RandomAccessRange  |                                       | *preserved*                                        |
-     * | std::ranges::ContiguousRange    |                                       | *lost*                                             |
-     * |                                 |                                       |                                                    |
-     * | std::ranges::ViewableRange      | *required*                            | *guaranteed*                                       |
-     * | std::ranges::View               |                                       | *guaranteed*                                       |
-     * | std::ranges::SizedRange         |                                       | *preserved*                                        |
-     * | std::ranges::CommonRange        |                                       | *preserved*                                        |
-     * | std::ranges::OutputRange        |                                       | *lost*                                             |
-     * | seqan3::ConstIterableRange      |                                       | *preserved*                                        |
-     * |                                 |                                       |                                                    |
-     * | seqan3::reference_t             | seqan3::Semialphabet                  | std::size_t                                        |
-     *
-     * See the \link view view submodule documentation \endlink for detailed descriptions of the view properties.
-     *
-     * ### Example
-     * \snippet test/snippet/range/view/kmer_hash.cpp usage
-     * \hideinitializer
-     */
-    inline auto constexpr kmer_hash = detail::kmer_hash_fn{};
+/*!\name Alphabet related views
```

Comment:
This is from the old kmer_hash_view documentation ..  and is it kinda alphabet related

Reply:
Does it depend on the value type being an Alphabet or does it work on anything hashable?

Reply:
Actually on anything hashable since I only do arithmetic coding for which i just need a representation as unsigned integer.
Do we want a `Hashable` concept or should I just check that `std::hash`'s `operator()` can be called on the type?

Reply:
**But** I need `alphabet_size`. So it's probably Semialphabet after all?

Reply:
Well, it's a design decision you have to make. I guess we can restrict it to semi-alphabets for now and still change it later. Please document this requirement clearly in the view interface!